packages/toolkit/language-kit/src/const.ts

```diff
@@ -32,6 +32,7 @@ export const languages = Object.freeze({
   'eu-ES': 'Euskara',
   'fa-IR': 'فارسی',
   'ff-NG': 'Fula',
+  fi: 'Suomi',
```

Comment:
just curious: will it affect console custom languages?

Reply:
Yes, it will, but in a good way, I think. As I'm only adding new language tags, the Console users will just have more options in the language dropdown.

<img width="228" alt="image" src="https://github.com/user-attachments/assets/85f6b07b-2469-4ca1-8bbe-8ba117a7b365" />